app/javascript/components/form_fields/TextField.spec.tsx

```diff
@@ -43,7 +43,7 @@ describe('TextField', () => {
 
 		it('is valid when value is changed and then deleted', async () => {
 			expect.assertions(1);
-			await userEvent.type(emptyTextField, "enter some information{selectall}{backspace}");
```

Comment:
Let's keep the await in. We have so many issues with inconsistent results on these tests when we don't use await.

Alternatively: figure out how to get the results to be more consistent :smile:

Reply:
sorry, we're back to 'await'. we removed await because we saw in the react testing library documentation that userEvent is a synchronous call, we could not reproduce the inconsistency.

Reply:
The error is pretty intermittent during the test. It's very frustrating.